tasks/section_4/cis_4.3.x.yml

```diff
@@ -17,9 +17,9 @@
 - name: "4.3.2 | PATCH | Ensure sudo commands use pty"
   ansible.builtin.lineinfile:
       path: /etc/sudoers
-      regexp: '^Defaults        use_'
-      line: 'Defaults        use_pty'
-      insertafter: '^Defaults'
+      regexp: '^\s*Defaults\s+use_pty\s*$'
+      line: 'Defaults use_pty'
+      insertafter: 'EOF'
```

Comment:
Should insert at end of file, in particular the inserted directive should take precedence over "Defaults !use_pty" if that appears in an included file or in #includedir.